docs/en/observability/apm/data-model.asciidoc

```diff
@@ -357,9 +357,9 @@ You can filter and group by these dimensions (some of which are optional, for ex
 * `host.name`: The user-defined name of the host or the detected hostname of the service that served the transaction
 * `host.os.platform`: The platform name of the service that served the transaction, for example `linux`
 * `kubernetes.pod.name`: The name of the Kubernetes pod running the service that served the transaction
-* `labels`: Key-value object containing string labels set globally by the APM agents.
+* `labels`: Key-value object containing string labels set globally by the APM agents except RUM agents.
```

Comment:
How can we make more evident to the reader that these data sent by RUM agents are discarded?
Is this the right place to add such note?

Reply:
Would it make sense to add another sentence, something like: "String labels set globally by RUM agents are discarded." (APM isn't my area, so I am just spitballing in an attempt to be helpful.)

Reply:
I gave it a go. Let's see if it looks better.